
remove the bogus hack that effectively ignores the --localstatedir option

--- configure.orig	Sat Aug  8 13:57:15 1998
+++ configure	Sat Aug  8 13:58:00 1998
@@ -573,11 +573,11 @@
 REGEXLIB=''	# -lregex
 LIBREGEX=''	# libregex.a
 
-if test "$libexecdir" = '${exec_prefix}/libexec'; then
-	libexecdir='${bindir}'
-	localstatedir='${prefix}'
-
-fi
+#if test "$libexecdir" = '${exec_prefix}/libexec'; then
+#	libexecdir='${bindir}'
+#	localstatedir='${prefix}'
+#
+#fi
 
 if test -z "$CACHE_HTTP_PORT"; then
 	CACHE_HTTP_PORT="3128"
--- configure.in.orig	Sat Aug  8 13:57:35 1998
+++ configure.in	Wed Mar 18 10:11:58 1998
@@ -20,11 +20,11 @@
 LIBREGEX=''	# libregex.a
 
 dnl use directory structure of cached as default (hack)
-if test "$libexecdir" = '${exec_prefix}/libexec'; then
-	libexecdir='${bindir}'
-	localstatedir='${prefix}'
-
-fi
+#if test "$libexecdir" = '${exec_prefix}/libexec'; then
+#	libexecdir='${bindir}'
+#	localstatedir='${prefix}'
+#
+#fi
 
 if test -z "$CACHE_HTTP_PORT"; then
 	CACHE_HTTP_PORT="3128"
